README.md

```diff
@@ -9,44 +9,70 @@ IntelliJ it is recommended to download and activate the
 [google-java-format plugin](https://github.com/google/google-java-format).
 
 
-Building and running manually/locally
--------------------------------------
+### BUILDING AND RUNNING THE PROJECT
 
-Initialize your local postgresql database as described in datamodel/README.md, then
-```
-export db_hostname=localhost
-export DCSA_Version=0.7.4 #or whatever version is the right one
-```
-If running without auth0, disable it in src/main/resources/application.yaml
+> **[RECOMMENDED]** Set up a Github Personal Access Token (PAT) as mentioned [here](https://github.com/dcsaorg/DCSA-Core/blob/master/README.md#how-to-use-dcsa-core-packages), then skip to **step 3**.
+
+If you would like to build required DCSA packages individually, begin with step 1.
+
+1) Build **DCSA-Core** as described
+   in [DCSA-Core/README.md](https://github.com/dcsaorg/DCSA-Core/blob/master/README.md#to-build-manually-run), then
+
+2) Build **DCSA-Event-Core** as described
+   in [DCSA-Event-Core/README.md](https://github.com/dcsaorg/DCSA-Event-Core/blob/master/README.md#to-build-manually-run)
+   , then
+
+3) Clone **DCSA-EBL** (with ``--recurse-submodules`` option.) and Build using, ``mvn package``
+
+4) Initialize your local postgresql database as described
+   in [datamodel/README.md](https://github.com/dcsaorg/DCSA-Information-Model/blob/master/README.md) \
+   or If you have docker installed, you may skip this step and use the docker-compose command mentioned below to set it
+   up (This will initialize the application along with the database).
+
+5) Run application,
 
-Then build and run with
 ```
-mvn install:install-file -Dfile=../DCSA-Core/target/dcsa_core-$DCSA_Version.jar -DgroupId=org.dcsa -DartifactId=dcsa_core -Dversion=local-SNAPSHOT -Dpackaging=jar -DgeneratePom=true
-mvn spring-boot:run -Ddcsa.version=local-SNAPSHOT
+mvn spring-boot:run [options]
+
+options:
+ -Dspring-boot.run.arguments="--DB_HOSTNAME=localhost:5432 --AUTH0_ENABLED=false --LOG_LEVEL=DEBUG"
```

Comment:
db_hostname needs to be lowercase, see: https://github.com/dcsaorg/DCSA-BKG/issues/122